patroni/api.py

```diff
@@ -119,6 +119,16 @@ def __init__(self, request: Any,
         self.__start_time: float = 0.0
         self.path_query: Dict[str, List[str]] = {}
 
+    def version_string(self) -> str:
+        """Override the default version string to return the server header as specified in the configuration.
+
+        If the server header is not set, then it returns the default version string of the HTTP server.
+
+        :return: ``Server`` version string, which is either the server header or the default version string
+        from the BaseHTTPRequestHandler.
```

Comment:
It fails: https://github.com/patroni/patroni/actions/runs/15759073435/job/44424269301?pr=3384#step:6:1843